sdk/spring/azure-identity-spring/src/main/java/com/azure/spring/identity/SpringCredentialBuilderBase.java

```diff
@@ -39,9 +39,12 @@ private TokenCredential populateTokenCredential(String prefix, boolean createDef
         String tenantId = getPropertyValue(prefix, "tenant-id");
         String clientId = getPropertyValue(prefix, "client-id");
         String clientSecret = getPropertyValue(prefix, "client-secret");
+        AzureCloud azureCloud = getPropertyValue(AzureCloud.class, prefix, "cloud-name", AzureCloud.Azure);
```

Comment:
I'm really not a big fan of using this **cloud-name** as property name, wired and not clear